modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java

```diff
@@ -378,6 +396,32 @@ public void testFreeMarkerNotificationWithCommerceOrder() throws Exception {
 				TestPropsValues.getUserId()));
 	}
 
+	@Test
+	public void testFreeMarkerNotificationWithCustomService() throws Exception {
+		String body = LocalizationUtil.updateLocalization(
+			LocalizedMapUtil.getLocalizedMap(
+				HashMapBuilder.put(
+					LanguageUtil.getLanguageId(LocaleUtil.US),
+					"Test Service : ${objectFieldBusinessType}"
+				).build()),
+			null, "Body", LanguageUtil.getLanguageId(LocaleUtil.US));
+
+		int queueEntries = getNotificationQueueEntries().size();
+
+		executeNotificationObjectAction(
+			0,
+			_addNotificationTemplate(
+				body, NotificationTemplateConstants.EDITOR_TYPE_FREEMARKER,
+				Collections.singletonMap(
+					LocaleUtil.US, "[%CURRENT_USER_FIRST_NAME%]"),
+				false,
+				Collections.singletonMap(
+					LocaleUtil.US, user1.getEmailAddress())));
```

Comment:
@rijema reorder:

```
int notificationQueueEntriesSize = ...;

String body = ...;

executeNotificationObjectAction(..);

Assert.assertEquals(...);
```

Reply:
I removed it the body  and notificationQueueEntriesSize one